packages/emotion/src/getTheme.ts

```diff
@@ -81,11 +80,9 @@ const getTheme =
           'No theme provided for [InstUISettingsProvider], using default `canvas` theme.'
         )
       }
-      //TODO: replace cloneDeeps with native `structuredClone` API
-      //once we hit last 2 version browser support
-      currentTheme = cloneDeep(globalTheme || canvas)
+      currentTheme = globalTheme || canvas
     } else {
-      currentTheme = cloneDeep(ancestorTheme)
+      currentTheme = ancestorTheme
```

Comment:
These `cloneDeep`'s were unnecessary because we don't modify the source objects.